src/main/java/com/barterbay/app/config/SecurityConfiguration.java

```diff
@@ -1,18 +1,65 @@
 package com.barterbay.app.config;
 
+import com.barterbay.app.service.UserService;
+import com.barterbay.app.util.Constants;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
 
-//TODO BB-21
+@Configuration
```

Comment:
@Configuration - no need here since we have **@EnableWebSecurity**

Reply:
corrected